specification/api-tracing.md

```diff
@@ -370,7 +371,7 @@ A `Span` MUST have the ability to set attributes associated with it.
 
 An `Attribute` is defined by the following properties:
 
-- (Required) The attribute key, which must be a string.
+- (Required) The attribute key, which must be a non-`null` and non-empty string.
```

Comment:
Are we sure non-empty is a good idea? Sometimes information can be conveyed by an empty string.

Reply:
oops realized this is key not value :)